### PR TITLE
rephrasing the pubkey-messages to be technically correct

### PR DESCRIPTION
**Pull request by spookey**
I have the feeling this sentence was wrong phrased and therefore may cause confusion for users.

So I rephrased it..

### CODE REVIEW

File: docs/site-example/i18n/de.po [hunk 1]
```diff
@@ -18,11 +18,12 @@ msgstr ""
...
-"Dies ist der öffentliche Schlüssel deines Freifunk-Knotens. Erst nachdem "
-"er auf den Servern des Entenhausener Freifunk-Projektes eingetragen wurde, "
-"kann sich dein Knoten mit dem Entenhausener Mesh-VPN zu verbinden. Bitte "
-"schicke dazu diesen Schlüssel und den Namen deines Knotens "
-"(<em><%=hostname%></em>) an "
+"Dies ist der öffentliche Schlüssel deines Freifunk-Knotens. "
+"Dein Knoten kann sich entweder über andere Knoten in Reichweite "
+"oder per Mesh-VPN über die Gateway-Server mit dem Entenhausener "
+"Freifunk-Netz verbinden. Letzteres ist erst möglich, nachdem der "
+"Schlüssel auf den  Servern des Entenhausener Freifunk-Projektes eingetragen wurde. "
```
Comment (tcatm) [inline docs/site-example/i18n/de.po]: Hier ist ein Leerzeichen zuviel.

File: docs/site-example/i18n/en.po [hunk 2]
```diff
@@ -17,9 +17,10 @@ msgstr ""
 
 msgid "gluon-config-mode:pubkey"
 msgstr ""
-"This is your Freifunk node's public key. The node won't be able to "
-"connect to the mesh VPN until the key has been registered on the Freifunk "
-"Duckburg servers. "
+"This is the public key of your Freifunk node. "
+"Your Node may ether connect via other nodes within range or "
```
Comment (tcatm) [inline docs/site-example/i18n/en.po]: ether -> either, Node -> node

Wie wäre es mit: Your node may either connect to nearby nodes through wifi or to gateway servers via Mesh-VPN...

File: docs/site-example/i18n/en.po [hunk 3]
```diff
@@ -27,7 +28,7 @@ msgstr ""
 msgid "gluon-config-mode:reboot"
 msgstr ""
 "<p>The node is currently rebooting and will try to connect to other "
```
Comment (tcatm) [inline docs/site-example/i18n/en.po]: "other" würde ich hier ganz streichen

**Comment by spookey** [comment]
Habe meine Tippfehler korrigiert, und die Vorschläge mit übernommen.
(Das "other" aus dem Letzten Vorschlag kam nicht von mir, war dennoch so frei dies mit rauszunehmen.)

File: docs/site-example/i18n/en.po [hunk 4]
```diff
@@ -17,17 +17,18 @@ msgstr ""
 
 msgid "gluon-config-mode:pubkey"
 msgstr ""
-"This is your Freifunk node's public key. The node won't be able to "
-"connect to the mesh VPN until the key has been registered on the Freifunk "
-"Duckburg servers. "
+"This is the public key of your Freifunk node. "
+"Your node may either connect to nearby nodes through wifi "
+"or to gateway servers via Mesh-VPN. "
+"Last option is only possible after the key has been added to the Server. "
```
Comment (tcatm) [inline docs/site-example/i18n/en.po]: Wie wäre es mit: Connecting via Mesh-VPN requires the key to be enabled by your community.

**Comment by tcatm** [comment]
Sieht gut aus und kann gemergt werden.

File: docs/site-example/i18n/de.po [hunk 5]
```diff
@@ -18,11 +18,13 @@ msgstr ""
...
 msgstr ""
-"Dies ist der öffentliche Schlüssel deines Freifunk-Knotens. Erst nachdem "
-"er auf den Servern des Entenhausener Freifunk-Projektes eingetragen wurde, "
-"kann sich dein Knoten mit dem Entenhausener Mesh-VPN zu verbinden. Bitte "
-"schicke dazu diesen Schlüssel und den Namen deines Knotens "
-"(<em><%=hostname%></em>) an "
+"Dies ist der öffentliche Schlüssel deines Freifunk-Knotens. "
+"Dein Knoten kann sich entweder über andere Knoten in Reichweite "
+"oder per Mesh-VPN über die Gateway-Server mit dem Freifunk-Netz "
+"verbinden. Um am Mesh-VPN teilnehmen zu können, muss muss der "
```
Comment (kokel) [inline docs/site-example/i18n/de.po]: Hier ist noch ein "muss" zuviel ...

**Comment by NeoRaider** [comment]
* Before this is merged, the commits should be squashed
* The English version has some issues ("Mesh-VPN" should stay "mesh VPN")
* I don't like using the term "gateway servers", as the VPN server are not always gateways
* I think we should avoid the word "wifi", "WLAN" is more correct

**Comment by rubo77** [comment]
This sounds somehow cheap still: 

>Your node's setup is now complete.
>
>The node is currently rebooting and will try to connect to other nearby Freifunk nodes after that. Your can find lots of information on the Freifunk Kiel community on our homepage.
>
>Have fun with your node and exploring the Freifunk network!

German wording sounds better:
```
"<p>Dein Knoten startet gerade neu und wird anschließend versuchen, "
"sich mit anderen Freifunkknoten in seiner Nähe zu "
"verbinden. Weitere Informationen zur "
"... Freifunk-Community findest du auf "
"<a href=\".../\">unserer Webseite</a>.</p>"
"<p>Viel Spaß mit deinem Knoten und der Erkundung von Freifunk!</p>"
```

Außerdem fehlt der Text übersetzt: 
```
"<p>Um zu dieser Konfigurationsseite zurückzugelangen, drücke im normalen "
"Betrieb für drei Sekunden den Reset-Button. Das Gerät wird dann im Config "
"Mode neustarten.</p>"```

**Comment by NeoRaider** [comment]
@spookey, what's the state of this PR? Will the issues I mentioned in my last comment be fixed?

**Comment by spookey** [comment]
No.

**Comment by NeoRaider** [comment]
This PR is dead, closing.

**Pull request closed without merging**